vector_dbs/milvus/Makefile

```diff
@@ -0,0 +1,55 @@
+REGISTRY ?= quay.io
+REGISTRY_ORG ?= ai-lab
+COMPONENT = vector_dbs
+
+IMAGE ?= $(REGISTRY)/$(REGISTRY_ORG)/$(COMPONENT)/milvus:latest
+
+ARCH ?= $(shell uname -m)
+PLATFORM ?= linux/$(ARCH)
+
+gRCP_PORT := 19530
+REST_PORT := 9091
+CLIENT_PORT := 2379
+
+LIB_MILVUS_DIR_MOUNTPATH := $(shell pwd)/volumes/milvus
+
+.PHONY: build
+build:
+	podman build --platform $(PLATFORM) -f Containerfile -t ${IMAGE} .
+
+.PHONY: run
+run:
+	podman run -d \
```

Comment:
@Shreyanand I pulled this from your example, but the upstream script runs with `sudo`. I cannot find any reference to why it was put there / if it is actually required. Can you confirm that it ran worked as a standard user?

Reply:
Also @Shreyanand please sign your commits. I set DCO to pass just to see if that was blocking RAG tests from running but no dice there. Until then /hold

Reply:
@Gregory-Pereira I tried it without sudo and it worked on my local. I am updating the PR with the reviews and DCO checks.